shared/whats-new/versions.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react'
+import * as Kb from '../common-adapters'
+import * as Styles from '../styles'
+import {accountTab, displayTab, contactsTab} from '../constants/settings'
+import NewFeatureRow from './new-feature-row'
+
+/* Include images */
+/* const imageName = require('../images/release/MAJ.MIN.PATCH/name.png') */
+const pinnedMessageImage = require('../images/releases/4.7.0/pinned-message.png')
+const darkModeImage = require('../images/releases/4.7.0/dark-mode.png')
+
+export type VersionProps = {
+  seen: boolean
+  onNavigate: (props: {}, selected: string) => void
+  onNavigateExternal: (url: string) => void
+}
+
+export const Version = ({children}: {children: React.ReactNode}) => {
+  return (
+    // Always pass `seen` prop to children of a version to show row-level badging
+    <Kb.Box2 direction="vertical" alignItems="flex-start" fullWidth={true}>
+      {children}
+    </Kb.Box2>
+  )
+}
+
+export const VersionTitle = ({title}: {title: string}) => (
+  <Kb.Box2 direction="vertical" alignItems="flex-start" fullWidth={true}>
+    <Kb.Text type="BodySemibold" style={styles.versionTitle}>
+      {title}
+    </Kb.Text>
+  </Kb.Box2>
+)
+
+export const Current = ({seen, onNavigate, onNavigateExternal}: VersionProps) => {
+  return (
+    <Version>
+      <NewFeatureRow
+        noSeparator={true}
+        seen={seen}
+        image={darkModeImage}
+        primaryButtonText="Go dark"
```

Comment:
If we ship this card, many people who already have dark mode on will see the "Go dark" button, which is a little confusing -- wonder if it should be "Open display settings".  Or, for bonus points, either "Go dark" or "Go light" based on which one isn't selected.  Or maybe this just doesn't matter.

Reply:
Good point. I'll make the change to `Open display settings`